### ISSUE TITLE
nknc info -v returns 0 when error

### ISSUE THREAD

**Issue by AGx10k** [nknorg/nkn#584]
### Description of the issue
IMHO `nknc info -v` (and other switches) should return non-0 exit code when error
```
# ./nknc info  -v
2019/07/09 15:40:57.727685 [ERROR] GID 1, POST request: Post http://localhost:30003: dial tcp [::1]:30003: connect: connection refused

Post http://localhost:30003: dial tcp [::1]:30003: connect: connection refused
# echo $?        
0
```

### Issue-Type (put a `x` sign in the square brackets)
- [X] bug report
- [X] feature request
- [ ] Documentation improvement

### Checklist
- [X] Running latest version of code.
- [X] This issue has not been reported earlier.

### Your environment
* nknc version v1.0.2-beta from release
* linux 64

### Expected behaviour
some non-0 exit code for better usage withing scripts (now need to parse output)

### Actual behaviour
0 is returned when error occurs

### Steps to reproduce
```
systemctl nkn stop
./nknc info -v
echo $?
```

### Any extra info ( for eg. code snippet to reproduce, logs, etc. )
If necessary, provide some extra information like code-snippets or error-logs.

**Comment by miserableoracle** [comment]
I think $? is about whether previous shell command executed successfully or not.

In this case, ./nknc call was successful because it performed what it was supposed to (sending a request in this case), the failure was received as POST response. 
Now if it was a failure at ./nknc part, ex. it couldn't send request due to some reason, it might have thrown an error code as it wasn't successful in doing what it was supposed to..

This is what I think.. We can discuss this more if you'd like. Also, please correct me if I'm wrong.

**Comment by AGx10k** [comment]
@miserableoracle that's exactly what i'm talking about - `echo $?` prints exit code of last command, and when nknc can not connect to node it should return non-zero exit code because it undoubtedly is an error

**Comment by AGx10k** [comment]
another example. trying to get state of some remote node (which is offline):
```
 $ ./nknc-v1.1.0-beta.1 --ip 1.1.1.1 info -s
2019/10/03 13:16:31.187855 [ERROR] GID 1, POST request: Post http://1.1.1.1:30003: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)

Post http://1.1.1.1:30003: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
 $  echo $?
0
```

**Comment by yilunzhang** [comment]
@NoelBright I think we should address this issue and return the correct code in nknc

**Comment by yilunzhang** [comment]
@AGx10k Does #652 solve your problem?

**Comment by yilunzhang** [comment]
Closing for now since it should be solved by #652. Feel free to re-open it if it's not working.

**Comment by AGx10k** [comment]
@yilunzhang thanks for the fix. it works. hope to see it in next release!
